src/hotspot/share/gc/g1/g1CollectedHeap.cpp

```diff
@@ -4568,6 +4574,10 @@ void G1CollectedHeap::set_used(size_t bytes) {
   _summary_bytes_used = bytes;
 }
 
+void G1CollectedHeap::set_live(size_t bytes) {
+  Atomic::release_store(&_live_size, bytes);
```

Comment:
I don't think this requires `release_store`, regular `store` would be enough. G1 folks can say for sure.

Reply:
👍